descriptors/ModuleDescriptor-template.json

```diff
@@ -52,6 +52,21 @@
           "methods": [ "POST" ],
           "pathPattern" : "/authn/password/repeatable",
           "permissionsRequired" : [ "login.password.validate" ]
+        },
```

Comment:
Update interface version "version": "4.3"
You add new endpoints so should update api version

Reply:
Thanks for the comments!
Done